gate-core/src/main/java/com/netflix/spinnaker/gate/services/TaskService.java

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2020 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.spinnaker.gate.services;
+
+import com.netflix.spinnaker.gate.services.commands.HystrixFactory;
+import com.netflix.spinnaker.gate.services.internal.ClouddriverServiceSelector;
+import com.netflix.spinnaker.gate.services.internal.OrcaServiceSelector;
+import com.netflix.spinnaker.security.AuthenticatedRequest;
+import java.util.*;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+@Service
+public class TaskService {
+
+  private final Logger log = LoggerFactory.getLogger(getClass());
+  private static final String GROUP = "tasks";
+
+  @Autowired private OrcaServiceSelector orcaServiceSelector;
```

Comment:
May as well clean up convention while we’re here: Use constructor injection instead.